### PR TITLE
added new info text to Buenos Aires Gathering

### PR DESCRIPTION
**Pull request by dmueller2001**
Replaced the english text with spanish

@mbetanco @wgordon17 @jiri-fiala

### CODE REVIEW

File: data/gatherings.yml [hunk 1]
```diff
@@ -20,9 +20,7 @@ gatherings:
...
-      The OpenShift Commons Gathering brings together experts from all over the world to discuss container technologies, best practices for cloud native application developers and
-      the open source software projects that underpin the OpenShift ecosystem. The Seattle event will gather developers, devops professionals and sysadmins together to explore
-      the next steps in making container technologies successful and secure.
+      El OpenShift Commons Gathering tendrá como eje las charlas a cargo de usuarios finales sobre implementaciones de OpenShift en modo de producción, en las que compartirán sus casos de uso, aprendizajes y buenas prácticas con expertos de todas partes del mundo. También analizará las tecnologías de contenedores, las buenas prácticas del desarrollo de aplicaciones nativas de la nube y los proyectos de software open source que apuntalan al ecosistema OpenShift para llevar a OpenShift al siguiente nivel del cloud computing nativo. El evento reunirá a desarrolladores, profesionales de DevOps y administradores de sistemas que explorarán y colaborarán para lograr que OpenShift y las tecnologías de contenedores sean exitosas y seguras a gran escala
```
Comment (wgordon17) [inline data/gatherings.yml]: I'm assuming this should end with a `.`. Also, would you be able to break this up into multiple lines, just to cut down on the amount of horizontal scrolling.

Comment (dmueller2001) [reply data/gatherings.yml]: what is the optimal length of the line of text? @wgordon17

Comment (wgordon17) [reply data/gatherings.yml]: For consistency, I would say try to keep it <180 characters wide

Comment (wgordon17) [reply data/gatherings.yml]: This is currently 752 characters wide ;)

**Pull request closed without merging**